rasa/shared/core/domain.py

```diff
@@ -85,17 +97,24 @@ class ActionNotFoundException(ValueError, RasaException):
 
 
 class SessionConfig(NamedTuple):
+    """The Session Configuration."""
+
     session_expiration_time: float  # in minutes
     carry_over_slots: bool
 
     @staticmethod
     def default() -> "SessionConfig":
+        """Returns the SessionConfig with the default values."""
         return SessionConfig(
-            rasa.shared.constants.DEFAULT_SESSION_EXPIRATION_TIME_IN_MINUTES,
-            rasa.shared.constants.DEFAULT_CARRY_OVER_SLOTS_TO_NEW_SESSION,
+            DEFAULT_SESSION_EXPIRATION_TIME_IN_MINUTES,
+            DEFAULT_CARRY_OVER_SLOTS_TO_NEW_SESSION,
         )
 
     def are_sessions_enabled(self) -> bool:
+        """Returns a boolean value depending on the value of session_expiration_time.
+
+        Returns True if the value of the session_expiration_time variable is greater
+        than zero and False if session_expiration_time is less than/equal to zero."""
```

Comment:
I think the code is clear enough and more concise than the extra docstring
```suggestion
"""
```